packages/flutter/test/material/scaffold_test.dart

```diff
@@ -1676,6 +1676,90 @@ void main() {
     await tester.pumpWidget(buildFrame(false, null));
     expect(tester.getSize(find.byKey(bodyKey)), const Size(800.0, 500.0));
   });
+
+  group('Throws FlutterError', () {
+    testWidgets(
+        'Call to Scaffold.of() without context', (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(
+          home: Builder(
```

Comment:
flutter style would be to put MaterialApp on its own line and put trailing commas for every block that could wrap.